config/environments/test.rb

```diff
@@ -36,5 +36,6 @@
   config.active_support.deprecation = :stderr
 
   # Raises error for missing translations
-  config.action_view.raise_on_missing_translations = true
+  # Temporarily setting to false because spotlight has some missing keys that need to be fixed
+  config.action_view.raise_on_missing_translations = false
```

Comment:
@jkeck is this ticketed anywhere?

Reply:
Ah, no, I was going to submit a PR (I have a local branch fixing some of them) but I bailed to get this out the door.  I'll add that ticket now. 👍